backend/prompt_studio/prompt_studio_index_manager/views.py

```diff
@@ -25,3 +34,54 @@ def get_queryset(self) -> Optional[QuerySet]:
         if filter_args:
             queryset = IndexManager.objects.filter(**filter_args)
         return queryset
+
+    def get_indexed_data_for_profile(self, request) -> Response:
+        queryset = self.get_queryset()
+        org_id = UserSessionUtils.get_organization_id(request)
+        util = PromptIdeBaseTool(log_level=LogLevel.INFO, org_id=org_id)
+        result = {}
+
+        if queryset is not None:
+            # Perform operations on the queryset
+            # Example: iterate through queryset and print each object's id
+            for obj in queryset:
+                profile_manager_id = obj.profile_manager_id
+                document_name = str(obj.document_manager.document_name)
+                try:
+                    profile_manager = ProfileManager.objects.get(
+                        profile_id=profile_manager_id
+                    )
+                except ProfileManager.DoesNotExist:
+                    raise DefaultProfileError(
+                        f"ProfileManager with ID {profile_manager_id} does not exist."
+                    )
+                vector_db_id = str(profile_manager.vector_store.id)
+                embedding_model = str(profile_manager.embedding_model.id)
```

Comment:
replace variable  like embedding_model_id,embedding_id or embedding_instance_id etc..

Reply:
Got it.